tasks.py

```diff
@@ -5,7 +5,7 @@
 from invoke import task
 
 try:
-    import toml
+    import tomli
 except ImportError:
     sys.exit("Please make sure to `pip install toml` or enable the Poetry shell and run `poetry install`.")
```

Comment:
Should probably update the library name in error message too.
_(I'll commit this suggestion. We can always revert it if I'm off-base.)_

```suggestion
    sys.exit("Please make sure to `pip install tomli` or enable the Poetry shell and run `poetry install`.")
```